lib/actions-oclif/device/rm.ts

```diff
@@ -85,7 +84,7 @@ export default class DeviceRmCmd extends Command {
 		// Remove
 		for (const uuid of params.uuid.split(',')) {
```

Comment:
Looks like this could be `uuids` instead of `params.uuid.split(',')` (duplicate call to `split()`).

Reply:
oops yes